dotCMS/src/main/java/com/dotmarketing/db/DbConnectionFactory.java

```diff
@@ -503,6 +503,24 @@ public static int getDbVersion() {
         return version;
     }
 
+    /**
+     * Method to get the Database Full Version
+     * @return Database Version (Major.Minor)
+     */
+    public static float getDbFullVersion() {
+        try {
+            Connection con = getConnection();
+            DatabaseMetaData meta = con.getMetaData();
+            String version = "%d.%d";
+            version = String.format(version, meta.getDatabaseMajorVersion(), meta.getDatabaseMinorVersion());
+            return Float.parseFloat(version);
+        } catch (Exception e) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Catch a list of specific exception subtypes instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2221)